assets/stylesheets/main.css

```diff
@@ -0,0 +1,213 @@
+/*** Project Meta ***/
+
+/* line - 13 general */
+
+/* line - 28 header */
+
+/* line - 89 main content */
+
+/* line - 154 footer */
+
+/** GENERAL **/
+
+html,
+body {
+  background-color: white;
+  position: relative;
+  margin: 0;
+  padding: 0;
+  font-size: 75%;
+}
+
+.font {
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+}
+
+/** header **/
+
+.header {
+  max-width: 78rem;
+  width: 100%;
+  margin: auto;
+  font-size: 1rem;
+  margin-top: 7rem;
+}
+
+.menu {
+  display: flex;
+  text-align: center;
+  justify-content: center;
```

Comment:
These styles are getting repeated. Avoid repeating styles. Create a class and add them there to reuse.